src/Pages/Auth/Login/index.js

```diff
@@ -21,12 +26,37 @@ function Login(props) {
     history.push("/login")
   }
 
+  const [formData, setFormData] = useState({ email: "", password: "" })
+
+  function handleSubmit() {
+    console.log(formData)
+  }
+
   return (
     <div>
-      <h1>This will be the Login page</h1>
+      <h3>{`${t("int.login")}`}</h3>
+      <form>
+        <label>
+          Email
+          <Input
+            name="email"
+            type="email"
+            onChange={(value) => setFormData({ ...formData, email: value })}
+          />
+        </label>
+        <label>
+          Password
+          <Input
+            onChange={(value) => setFormData({ ...formData, password: value })}
+            type="password"
```

Comment:
we should pass value down to the input

Reply:
if i add a value property in the Input component, the input doesnt accept anything to type in. It just displays a placeholder "true"